src/styles/_page.scss

```diff
@@ -0,0 +1,13 @@
+.page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  margin: 0;
+
+  font: {
+    family: $font;
+    size: $font-size;
+  }
```

Comment:
The use of the nested 'font' property is incorrect. In SCSS, you should use the shorthand 'font' property directly or specify 'font-family' and 'font-size' separately. The current syntax is not valid CSS.

Reply:
this is scss syntax